content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,115 @@
+---
+title:       "Newsletter 102 - 2022/2023 news"
+author:      "GeoB99"
+date:        2023-03-25
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers and enthusiasts! It has been quite a long time since we published the [101th newsletter](/blogs/newsletter-101/) and so far no further updates have been posted since then.
+While the [ReactOS Twitter account](https://twitter.com/reactos) does provide announcements, posts about working applications and such from time to time, much of what is happening with the project as a whole isn't mentioned. Most of what we are going to talk about is the current situation with releases and overall ReactOS development.
+
+## Why does it take so long to release a new version of ReactOS? Are you guys dead?
+The latest release of ReactOS is 0.4.14, published on 16th of December, 2021. That release alone took a year to be released. Back then the ReactOS followed the 3-months cadence, which every 3 months a new release would be released to the public. But since 2021, ReactOS is still at 0.4.14. Are you guys dead?
+
+The answer is certainly no, the way we handle releases has changed. Back in the day the way ReactOS releases were made were done for the sake of quantity rather than quality. Every new release after 3 months is similar to the last one, with the exceptions of a small number of features and bug fixes added to the release. But overall the net gap between the releases wasn't that big.
```

Comment:
```suggestion
The answer is certainly no, the way we handle releases has changed. Back in the day, ReactOS releases were made for the sake of quantity rather than quality. Every new release after 3 months was similar to the previous one, with the exception of a small number of added features and bug fixes. But overall, the net gap between the releases wasn't that big.
```